src/qt/nametablemodel.cpp

```diff
@@ -49,6 +49,7 @@ class NameTablePriv
     /* Local cache of name table.
      */
     QList<NameTableEntry> cachedNameTable;
+    std::map< std::string, NameTableEntry > cachedNameMap;
```

Comment:
Nit: Please remove the spaces around the template arguments (as in e.g. the line just above).